### DISCUSSION TITLE
Some Colours not customizable

### DISCUSSION THREAD

**Discussion by LTKort** [laravel/nova-issues#4207] [Ideas & Feature Requests]
Hi,

In the issue #4206 I proposed a fix to give the option to change the input border etc. Which looks okay:
![Screenshot 2022-05-10 at 14 26 39](https://user-images.githubusercontent.com/2412670/167628315-957c09c7-19d4-4dcc-80e7-ecf9da15f26b.png)

But some elements stay default...

Like the checkbox;
![Screenshot 2022-05-10 at 14 27 22](https://user-images.githubusercontent.com/2412670/167628429-e5e9eaa5-9e33-463d-8958-5f08046f3f20.png)

Or the focus of a form element / button:
![Screenshot 2022-05-10 at 14 26 58](https://user-images.githubusercontent.com/2412670/167628385-6128856b-74fc-45f7-944e-f100f710c9ee.png)

Is there an easy way to fix this?

**Comment by davidhemphill** [comment]
I've gone through and specified our rings to use the `primary-200`. Should come out in the next release. This will make them follow the branding colors.

**Comment by davidhemphill** [comment]
Ok, the newest release includes this. Please let me know if there's any part of Nova's UI I missed that's not displaying the correct colors.

**Comment by LTKort** [comment]
@davidhemphill  Thanks!

Is it possible to add a hover color to the icons on the ResourceTableRow or just something like;
```csss
.toolbar-button {
  @apply flex items-center justify-center flex-shrink-0 rounded-lg h-9 focus:ring ring-primary-200 ring-inset focus:outline-none;
  @apply hover:text-primary-400;
  @apply active:text-primary-600;
}
```

So it's something like:
![Screenshot 2022-05-11 at 14 21 56](https://user-images.githubusercontent.com/2412670/167848681-b4b73a20-6b10-48bb-9e17-8b516c872e87.png)

**Reply by davidhemphill** [reply]
You could use an asset to overwrite that class if you'd like. We added a hover state but didn't want to make it a primary color.

**Reply by LTKort** [reply]
I see a 'hover:opacity-50' on the header & notification icons, but not on the ResourceTableRow buttons. Or is it just me?

**Reply by davidhemphill** [reply]
Yeah those have been removed in this update to make them more consistent across the app.

**Reply by davidhemphill** [reply]
Went ahead and added a primary hover color to the view/edit/delete icons. Hope that helps!

**Comment by LTKort** [comment]
It looks like the loading bar is still the primary blue too

![Screenshot 2022-05-11 at 15 47 28](https://user-images.githubusercontent.com/2412670/167865292-601b8a5d-423b-4d55-a1e6-ddfbbe76ee3a.png)

**Reply by davidhemphill** [reply]
Fixed. Will push this soon.

**Comment by RhysLees** [comment]
@LTKort Unrelated but what's the package called that you used for the tabs? thanks
 :)

**Reply by LTKort** [reply]
Do you mean the main / SEO / Advanced tabs? 
That’s this one https://github.com/eminiarts/nova-tabs

**Reply by RhysLees** [reply]
> Do you mean the main / SEO / Advanced tabs? 
> That’s this one https://github.com/eminiarts/nova-tabs

Perfect, thanks!
